resize/index.html

```diff
@@ -70,32 +78,46 @@
         target.style.transform = `translate(${x}px, ${y}px)`;
         target.setAttribute('data-x', x);
         target.setAttribute('data-y', y);
+        localStorage.setItem('x', x.toString());
+        localStorage.setItem('y', y.toString());
       }
 
       function enableInteractives(hb) {
-        lock.classList.remove("hidden");
-        unlock.classList.add("hidden");
+        lock.classList.remove('hidden');
+        unlock.classList.add('hidden');
         interact('.resize-drag')
           .resizable({
             edges: { left: true, right: true, bottom: true, top: true },
             listeners: {
               move (event) {
                 const target = event.target;
+                const width = Math.floor(event.rect.width);
+                const height = Math.floor(event.rect.height);
                 let x = (parseFloat(target.getAttribute('data-x')) || 0);
                 let y = (parseFloat(target.getAttribute('data-y')) || 0);
 
-                let width = Math.floor(event.rect.width)
-                let height = Math.floor(event.rect.height)
-                if (width * height <= hb.initialWidth * hb.initialHeight) {
-                  hb.resize(width, height)
-                }
-
                 x += event.deltaRect.left;
                 y += event.deltaRect.top;
 
+                target.style.width = `${width}px`;
```

Comment:
avoid setting width, it will break if you have two tabs open

Reply:
We set width to "preview" the new dimensions before the user lets go of the mouse.
We also set width initially to load the container before the Hyperbeam virtual browser has loaded.